django_celery_beat/schedulers.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, model, app=None):
             self.options['expires'] = getattr(model, 'expires_')
 
         self.options['headers'] = loads(model.headers or '{}')
+        self.options['periodic_task_name'] = model.name
```

Comment:
no model change needed?

Reply:
yes, `model.name` is the name of PeriodicTask

Reply:
tests docs, model fields changes?

Reply:
Actually, docs and tests is missing , but no model changes required. I'll handle it.

Reply:
ok

Reply:
when you have time can you try https://github.com/celery/django-celery-beat/pull/464 PR and see if it's working correctly?

Reply:
Okay, I'm glad to do it these days. test #464 and #477 and celery/django-celery-results#261